cartridges/int_cloudinary/cartridge/templates/default/include/cloudinaryProductScripts.isml

```diff
@@ -0,0 +1,12 @@
+
+<isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled && (pdict.cloudinary.galleryEnabled || pdict.cloudinary.isGalleryEnabled)}">
+    <isset name="cloudinaryConstants" value="${require('*/cartridge/scripts/util/cloudinaryConstants')}" scope="page" />
+    <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_VIDEO_RESOURCE_TYPE + '/list/'+ pdict.product.CLDTagName +'.json'}" crossorigin as="fetch" />
+    <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_IMAGE_RESOURCE_TYPE + '/list/'+ pdict.product.CLDTagName +'.json'}" crossorigin as="fetch" />
+    <isif condition="${CloudinaryConstants.CLD_360_SPINSETS_ENABLED}">
+        <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_SPIN_SET_RESOURCE_TYPE + '/list/'+ pdict.product.CLDTagName + CloudinaryConstants.CLD_360_SPIN_SET_TAG_SUFFIX +'.json'}" crossorigin as="fetch" />
```

Comment:
@asad-rafter please use CLD_IMAGE_RESOURCE_TYPE (instead of CLD_SPIN_SET_RESOURCE_TYPE)